eval/eval_test.go

```diff
@@ -11,6 +11,13 @@ import (
 	"grol.io/grol/parser"
 )
 
+func init() {
+	err := extensions.Init()
+	if err != nil {
+		panic(err)
+	}
+}
```

Comment:
Nope


```suggestion
func TestMain(m *testing.M) {	
	err := extensions.Init()
	if err != nil {
		panic(err)
	}
	os.Exit(m.Run())
}
```

Reply:
I realize you don't like init() (for good reasons in library or main code) but how is that necessary or different or better for a test package

Reply:
Here it's not only because init, it's because there is something made to prepend things in tests

Reply:
I did anyway :) thx